spec/polaris-catalog-service.yaml

```diff
@@ -17,6 +17,8 @@
 # under the License.
 #
 
+# CODE_COPIED_TO_POLARIS
```

Comment:
Is this file really copied from Iceberg? It looks like a genuine Polaris source file to me :thinking: